engine/crates/engine-v2/src/sources/graphql/request/execute.rs

```diff
@@ -36,6 +39,8 @@ pub(crate) async fn execute_subgraph_request<'ctx, 'a, R: Runtime>(
 ) -> ExecutionResult<SubgraphResponse> {
     let endpoint = ctx.endpoint();
 
+    let http_span = SubgraphHttpRequestSpan::new(endpoint.url(), &http::Method::POST);
```

Comment:
we're already creating that span a bit later: `    let http_span = ctx.create_subgraph_request_span(&request);
`

Reply:
ah we should move the propagator later after the request creation